RHEcosystemAppEng/dbaas-operator
# Update the code to install SBO using OLM Dependency. Fix SBO Binding Permission issue

## Description

Change the approach of SBO installation using OLM Dependency see the file dependencies.yaml within bundle folder.
updated the version of SBO to use : ">=1.0.0"


Fix the below-seen permission issue of the Service binding operator into the DBaaS install namespace by adding RoleBinding which refers to service-binding-controller-role 

![Screen Shot 2022-01-17 at 3 50 15 PM](https://user-images.githubusercontent.com/41960152/149838583-456a2a74-8919-479b-99b4-db07df159821.png)


updated SBO labels as per the latest operator version guideline. 

Jira issue:  
https://issues.redhat.com/browse/DBAAS-255  

## Verification Steps

1. Check out the PR code and make an image and install the DBaaS operator.
2. Make sure the SBO operator get installed successfully 
3. Create DBaaS connection
4. Install sample application for Crunchy or MongoDB based on the connection provider type.
5. Connect the application with connection from topology view
6. Check if the application successfully connected with the selected database.

## Type of change


## Checklist (Definition of Done)
